packages/components/src/toolbar/test/index.js

```diff
@@ -14,8 +14,8 @@ describe( 'Toolbar', () => {
 		it( 'should render a toolbar with toolbar buttons', () => {
 			const wrapper = mount(
 				<Toolbar __experimentalAccessibilityLabel="blocks">
-					<ToolbarButton title="control1" />
-					<ToolbarButton title="control2" />
+					<ToolbarButton label="control1" />
+					<ToolbarButton label="control2" />
```

Comment:
When used within `<Toolbar __experimentalAccessibilityLabel>`, `ToolbarButton` passes all the props down to `Button` so they have the same API.

Reply:
Nice 👍